Tracers.pyx

```diff
@@ -205,6 +205,13 @@ cdef class UpdraftTracers:
         NS.add_profile('env_thetarho', Gr, Pa, units=r'K', nice_name=r'\theta_{\rho,e}',
                        desc=r'environment density potential temperature')
 
+        NS.add_profile('updraft_density_perturbation_pressure', Gr, Pa, units=r'm^2 s^-2 ', nice_name=r'updraft alpha0 * dynamic pressure',
+                      desc=r'updraft alpha0 * dynamic pressure')
```

Comment:
Can we change dynamic pressure to perturbation pressure or something else? Dynamic pressure is also the name of \rho u^2/2.